content/reference/drops/contact_request_drop.md

```diff
@@ -0,0 +1,25 @@
+# Contact Request Drop
+
+Allows to check and show Contact Form errors.
```

Comment:
We should decide to either use `*XXX Drop*` at the start of this sentence or not